dockstore-client/pom.xml

```diff
@@ -48,14 +48,6 @@
                     <groupId>org.postgresql</groupId>
                     <artifactId>postgresql</artifactId>
                 </exclusion>
-                <exclusion>
```

Comment:
Might be explained later, but instead of excluding would seem to be better to delete outright from dockstore-common

Reply:
I'm not excluding it, I'm removing the exclusion because it's not there in the first place anymore